gtsfm/data_association/data_assoc.py

```diff
@@ -115,9 +122,12 @@ def run(
         exit_codes_wrt_computed: List[TriangulationExitCode] = []
         per_accepted_track_avg_errors = []
         per_rejected_track_avg_errors = []
-        for track_2d in tracks_2d:
+        assert len(tracks_2d) == len(sfm_tracks)
+        for j in range(len(tracks_2d)):
```

Comment:
nit: since we don't triangulate anymore here, I believe the comment below needs to be updated

Reply:
Done